static/app/views/dashboardsV2/widgetLibrary/data.tsx

```diff
@@ -9,109 +9,136 @@ export type WidgetTemplate = Widget & {
 export const DEFAULT_WIDGETS: Readonly<Array<WidgetTemplate>> = [
   {
     id: undefined,
-    title: t('Total Errors'),
-    description: 'Total number of error events in a given time interval.',
-    displayType: DisplayType.BIG_NUMBER,
+    title: t('Duration Distribution'),
+    description: t('Compare transaction durations across different percentiles.'),
+    displayType: DisplayType.AREA,
     widgetType: WidgetType.DISCOVER,
     interval: '5m',
     queries: [
       {
         name: '',
-        conditions: '!event.type:transaction',
-        fields: ['count()'],
+        conditions: '!event.type:error',
+        fields: [
+          'p50(transaction.duration)',
+          'p75(transaction.duration)',
+          'p95(transaction.duration)',
+        ],
         orderby: '',
       },
     ],
   },
   {
     id: undefined,
-    title: t('All Events'),
-    description: 'Area chart reflecting all error and transaction events.',
-    displayType: DisplayType.AREA,
+    title: t('High Throughput Transactions'),
+    description: t('Top 5 transactions with the largest volume.'),
```

Comment:
I noticed when I added this widget on prod (through dev-ui), there were 5 and then the "Other" transactions kind of shrank everything. I was just curious if we had the ability to ignore the "Other" stuff. If possible, is there a chance we might want to do that here?

![Screen Shot 2021-12-08 at 1 44 42 PM](https://user-images.githubusercontent.com/22846452/145280119-51799cae-81d4-439f-8068-7356cfcbcaf6.png)

Reply:
It's possible to disable other by clicking on it in the legend. I kinda wish the "other" part of the graph was clickable or something so you could disable it easily...